packages/@sfx/core/test/unit/common/core.test.ts

```diff
@@ -0,0 +1,84 @@
+import { expect, sinon, spy, stub } from '../../utils';
```

Comment:
`spy` is not being used in the file and can be removed from the import. 
```suggestion
import { expect, sinon, stub } from '../../utils';
```